doc/chips/10.rst

```diff
@@ -340,14 +341,14 @@ initialization:
 	Child Phase 2
 
 
-The syntax to call a sibling initializer is ``this.init(<args>);``.  Similarly
-to in the parent-referential case, the return from that call will indicate that
-the calling initializer has begun Phase 2. In contrast to the parent case,
-however, a call to a sibling initializer may not be made if the current
-initializer has defined any fields.  The rationale for this decision is that a
-valid sibling initializer will initialize all of its fields, and so any
-initialization prior to that call will lead to the occurrence of an unexpected
-double initialization.
+The syntax to call another initializer defined on the same type is
+``this.init(<args>);``.  Similarly to in the parent-referential case, the return
+from that call will indicate that the calling initializer has begun Phase 2. In
+contrast to the parent case, however, a call to an outside initializer may not
+be made if the current initializer has defined any fields.  The rationale for
+this decision is that a valid alternate initializer will initialize all of its
```

Comment:
I once read a really nice piece of advice on technical documents. The suggestion was to stick to one set of terms (since you bother to define them carefully) and avoid using synonyms that would otherwise be nice English writing. Here too, "alternate" is a new term.